tdms/tests/unit/test_hdf5_io.cpp

```diff
@@ -18,9 +18,9 @@
 
 // tdms
 #include "arrays.h"
-#include "fdtd_grid_initialiser.h"
 #include "unit_test_utils.h"
 
+using namespace std;
```

Comment:
Not a strong comment, but you aren't really benefiting from using the namespace here. I mean: we don't make the code significantly shorter nor save that much whitespace (`std::` is only 5 characters) and you only delete the prefix in a couple of places.

Reply:
Yeah sorry, should've been more vigilant when cleaning up my code :sweat_smile: It's my own laziness for moving function names and the like around whilst trying to get MATLAB to play nicely with HDF5